distribution/lib/Standard/Database/0.0.0-dev/src/DB_Table.enso

```diff
@@ -174,6 +174,22 @@ type DB_Table
         col = self.get selector if_missing=Nothing
         if Nothing == col then if_missing else col.get index if_missing
 
+    ## ALIAS row
+       GROUP Standard.Base.Selections
+       ICON select_row
+       Gets a row from the table.
+
+       Arguments:
+       - index: The index of the row to get within the table.
+       - if_missing: The value to use if the selector isn't present.
+    @index (t-> Widget.Numeric_Input minimum=0 maximum=t.row_count-1)
```

Comment:
Using `row_count` will cause the DB backend to be running the underlying query (in a `COUNT(*)` mode).

I guess it's probably OK, but just noting this fact so that we do this consciously.

Reply:
It may introduce a greater delay to computing the widget to display, especially for remote DBs.

Reply:
Yes - we had this on get_value too, Think might be good to remove from widget.

Reply:
Indeed, maybe until caching is better we should only keep those on in-memory.

As definitely useful on in-memory and there the overhead is none.